tex/gregoriotex.lua

```diff
@@ -267,9 +267,23 @@ local function get_gregorioversion()
   return internalversion
 end
 
+local function map_fonts()
+  -- use greciliae because it is the most complete font
+  local font_info = fontloader.open(kpse.find_file('greciliae', 'truetype fonts'))
```

Comment:
(sorry for thinking about that after merging) Here it might be useful to 

    log("Mapping font %s", kpse.find_file('greciliae', 'truetype fonts'))

because there might be edge cases where luaotfload doesn't use the font returned by kpse.find_file... luaotfload also looks in fontcache, etc. Might ease debugging...